osu.Game.Tests/Visual/Editing/TestSceneEditorClipboard.cs

```diff
@@ -122,6 +122,8 @@ public void TestCutPasteSpinner()
         [TestCase(true)]
         public void TestCopyPaste(bool deselectAfterCopy)
         {
+            const int paste_time = 2000;
```

Comment:
You adjusted a test but didn't add explicit coverage of the snapping. Can you add a new test covering this please?